test/utils/test-utils.js

```diff
@@ -247,6 +247,17 @@ const normalizeError = (output) => {
     return output.replace(/SyntaxError: .+/, 'SyntaxError: <error-message>').replace(/\s+at .+(}|\)|\d)/gs, '\n    at stack');
 };
 
+const normalizeCompileTime = (output) => {
+    return output.replace(/in \d+ ms/gm, 'in <compile time> ms').replace();
+};
+
+const normalizeV4Output = (output) => {
+    return output
+        .replace(/Hash: .*/gm, 'Hash: <hash>')
+        .replace(/Time: .*/gm, 'Time: <compile time>')
+        .replace(/Built at: .*/gm, 'Built at: <built time>');
+};
```

Comment:
Put `normalizeCompileTime` in `normalizeV4Output`, ideally we should have two function `normalizeStatsV4` and `normalizeStatsV5`, v5 default output are simple so we should not have big problems here, If you need I can implement these functions so you will be able to continue adding snapshots and improve `normalizeStats` function using regexps

Reply:
yep help appreciated, also something wrong with v4 snapshot, thanks!